var/spack/repos/builtin/packages/sonlib/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Sonlib(MakefilePackage):
+    """sonLib is a compact C/Python library for
+    sequence analysis in bioinformatics."""
+
+    # This is sonlib as needed by the hal package which expects
+    # a side by side compilation
+    #
+    # If you need to use sonlib then you want py-sonlib
+
+    homepage = "https://github.com/ComparativeGenomicsToolkit"
+    url      = "https://github.com/ComparativeGenomicsToolkit/sonLib"
+    git      = "https://github.com/ComparativeGenomicsToolkit/sonLib.git"
+
+    version('devel', branch='master')
+    version('20200401', commit='7ebe2ede05a6ee366d93a7a993db69a99943a68f')
```

Comment:
```suggestion
    version('2020-04-01', commit='7ebe2ede05a6ee366d93a7a993db69a99943a68f')
```

Reply:
Fair point. I guess that it removes any ambiguity as to if the commit is from April 20200.

Reply:
It just makes version comparison a little more intuitive. For example, if you need a version from 2020 or newer, you can do:
```python
depends_on('sonlib@2020:')
```
instead of:
```python
depends_on('sonlib@20200000:')
```